### PR TITLE
Исправил перевод

### PR DESCRIPTION
**Pull request by nikitabelonogov**

### CODE REVIEW

**Comment by me-shaon** [comment]
@thecsw , Could you please review this PR?

File: translations/LICENSE_ru-RU [hunk 1]
```diff
@@ -2,20 +2,20 @@ GLWT(Good Luck With That) Общественная Лицензия
...
 Всем разрешается копировать, распространять, изменять, совмещать, продавать,
-публиковать, сублецинзировать или делать все, что угодно, но на СВОЙ СТРАХ и РИСК.
+публиковать, сублецинзировать или делать все что угодно на СВОЙ СТРАХ и РИСК.
 
-          	     УДАЧИ С ТОЙ ОБЩЕСТВЕННОЙ ЛИЦЕНЗИЕЙ
+                    УДАЧИ С ЭТОЙ ОБЩЕСТВЕННОЙ ЛИЦЕНЗИЕЙ
       УСЛОВИЯ И ПОЛОЖЕНИЯ ДЛЯ КОПИРОВАНИЯ, РАСПРОСТРАНЕНИЯ И ИЗМЕНЕНИЯ
 
-0. Вы просто ДЕЛАЕТЕ ВСЕ, ЧТО ВЫ ХОТИТЕ до тех пор, пока ВЫ НЕ ОСТАВЛЯЕТЕ УЛИК
+0. Вы можете ДЕЛАЕТЬ ВСЕ, ЧТО ЗАХОТИТЕ до тех пор, пока ВЫ НЕ ОСТАВЛЯЕТЕ УЛИК
```
Comment (thecsw) [inline translations/LICENSE_ru-RU]: `ДЕЛАТЬ` instead of `ДЕЛАЕТЬ`

File: translations/LICENSE_ru-RU [hunk 2]
```diff
@@ -2,20 +2,20 @@ GLWT(Good Luck With That) Общественная Лицензия
 Copyright (c) Все, кроме Автора
 
 Автор не имеет ни малейшего понятия, какую функцию этот код выполняет
-в проекте. Он может сработать или не сработать, третьего выбора не дано.
+в проекте. Он может сработать или не сработать, третьего не дано.
 
 Всем разрешается копировать, распространять, изменять, совмещать, продавать,
-публиковать, сублецинзировать или делать все, что угодно, но на СВОЙ СТРАХ и РИСК.
+публиковать, сублецинзировать или делать все что угодно на СВОЙ СТРАХ и РИСК.
```
Comment (thecsw) [inline translations/LICENSE_ru-RU]: There should be a comma!

`...или делать все, что угодно на СВОЙ...`

File: translations/LICENSE_ru-RU (hunk 1 above)
Comment (nikitabelonogov) [reply translations/LICENSE_ru-RU]: Yes, my bad

File: translations/LICENSE_ru-RU (hunk 2 above)
Comment (nikitabelonogov) [reply translations/LICENSE_ru-RU]: In this case, `все что угодно` act as an indissoluble combination and we can replace it with a single word eg. `абсолютно всё`
so a comma is not needed

**Comment by me-shaon** [comment]
@thecsw @nikitabelonogov please let me know when it is ready to merge.

**Comment by thecsw** [comment]
It's okay to merge.

**Pull request merged**